assets/global.js

```diff
@@ -599,7 +599,10 @@ class VariantSelects extends HTMLElement {
     if(this.stickyHeader) {
       this.stickyHeader.dispatchEvent(new Event('preventHeaderReveal'));
     }
-    window.setTimeout(() => { parent.querySelector('li.product__media-item').scrollIntoView({behavior: "smooth"}); });
+    window.setTimeout(() => {
+      parent.scrollLeft = 0;
+      parent.querySelector('li.product__media-item').scrollIntoView({behavior: 'smooth'});
```

Comment:
@ludoboludo Sorry for revisiting this, but if we are to keep `parent.scrollLeft = 0;` as a fallback, would it make sense to have it after `scrollIntoView()`? This way on mobile if the `behavior: 'smooth'` option is supported, the smooth scrolling will occur first and if it isn't supported, `parent.scrollLeft = 0` will kick in. Having it the opposite way prevents the `behavior: 'smooth'` from ever being effective?

```suggestion
      parent.querySelector('li.product__media-item').scrollIntoView({behavior: 'smooth'});
      parent.scrollLeft = 0;
```

Reply:
Nevermind, in my testing I'm not really seeing a difference 😅